rails_application/test/services/orders/submit_service_test.rb

```diff
@@ -13,9 +13,7 @@ def test_successful_order_submission
       prepare_product(product_id, "Async Remote", 49)
       run_command(Ordering::AddItemToBasket.new(order_id: order_id, product_id: product_id))
 
-      result = Orders::SubmitService.new(order_id: order_id, customer_id: customer_id).call
-
-      assert_equal result.status, :success
+      assert_equal true, Orders::SubmitService.new(order_id: order_id, customer_id: customer_id).call
```

Comment:
Let's try to test some actual effect of running this service object (read model value?) instead of testing it's return value

Reply:
Done, thank you for your comments! :)